tests/e2e-cypress/support/commands.js

```diff
@@ -74,6 +75,33 @@ const addSecondIconsFixtureAndIntercept = () => {
     }).as('icon-set-babs')
 }
 
+const addGeoJsonIntercept = () => {
+    cy.intercept('**/test.geojson.layer.json', {
+        crs: {
+            properties: { name: 'EPSG:2056' },
+        },
+        features: [
+            { type: 'Feature', geometry: { type: 'Point', coordinates: [2701702, 1265811] } },
+        ],
+        type: 'FeatureCollection',
+    }).as('geojson-data')
+    cy.intercept('**/vectorStyles/**', {
+        type: 'single',
+        geomType: 'point',
+        vectorOptions: {
+            type: 'circle',
+            radius: 9,
+            fill: {
+                color: '#00FFFF',
+            },
+            stroke: {
+                color: '#000000',
+                width: 1,
+            },
+        },
+    }).as('geojson-style')
+}
```

Comment:
Could you create fixture files for these two entries instead of hard coding them here? Might be easier to use them later on (to retrieve their data in the tests)

Reply:
done